### ISSUE TITLE
examples lde no longer works out of the box

### ISSUE THREAD

**Issue by ricksbrown** [BorderTech/wcomponents#1005]
As of the latest builds the self-executable wcomponent example jar give this error in the browser:

```
You need to set the class name of the WComponent you want to run.
Do this by setting the parameter "bordertech.wcomponents.lde.component.to.launch" in your "local_app.properties" file.
Eg. bordertech.wcomponents.lde.component.to.launch=com.github.bordertech.wcomponents.examples.picker.ExamplePicker
```

**Comment by jonathanaustin** [comment]
The `wcomponents-app.properties` file in `wcomponents-examples-lde` is overriding the `wcomponents-app.properties' file from `wcomponents-examples' when the standalone jar with dependencies is built.

To fix this the properties file should be deleted from `wcomponents-examples-lde`, as the properties it contains are already set.

**Comment by marksreeves** [comment]
Fixed in #1113
